<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>tiff</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gif</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ppm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bmp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>jpg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tga</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert To</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="dispPath">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>40</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgDisp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>311</width>
      <height>247</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="GetImage">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Image saved at:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dispLabel2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgDisp2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>311</width>
      <height>247</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="Sending">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert to grayscale</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>browse</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>getImageName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>send</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>sendImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GetImage</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>ReceiveImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getImageName()</slot>
  <slot>sendImage()</slot>
  <slot>ReceiveImage()</slot>
 </slots>
</ui>
